prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -379,7 +379,12 @@ if has_processing_step MUON_SYNC_RECO; then
   elif [[ $RUNTYPE == "PHYSICS" && $BEAMTYPE == "pp" || $LIGHTNUCLEI == "1" ]] || [[ $RUNTYPE == "COSMICS" ]]; then
     MCH_CONFIG_KEY+="MCHTracking.chamberResolutionX=0.4;MCHTracking.chamberResolutionY=0.4;MCHTracking.sigmaCutForTracking=7.;MCHTracking.sigmaCutForImprovement=6.;"
   fi
-  has_detector_reco ITS && [[ $RUNTYPE != "COSMICS" ]] && MCH_CONFIG_KEY+="MCHTimeClusterizer.irFramesOnly=true;"
+  if has_detector_reco ITS && [[ $RUNTYPE != "COSMICS" && x"${MCH_DISABLE_ITS_IRFRAMES_SELECTION:-}" != "x1" ]]; then
+    MCH_CONFIG_KEY+="MCHTimeClusterizer.irFramesOnly=true;"
+    [[ -z ${CUT_RANDOM_FRACTION_MCH:-} && -n ${CUT_RANDOM_FRACTION_MCH_WITH_ITS:-} ]] && CUT_RANDOM_FRACTION_MCH=${CUT_RANDOM_FRACTION_MCH_WITH_ITS:-}
+  else
```

Comment:
if you have ITS in cosmics, you are using `CUT_RANDOM_FRACTION_MCH_NO_ITS` here. Even though in this case by default at the moment, both `CUT_RANDOM_FRACTION_MCH_NO_ITS` and `CUT_RANDOM_FRACTION_MCH_WITH_ITS` are defined to be equal here https://github.com/AliceO2Group/O2DPG/blob/a53f4c0e5d7c504458aae055b546e4633e80d0f5/DATA/production/workflow-multiplicities.sh#L311, I suggest to adjust the logic so that `CUT_RANDOM_FRACTION_MCH_WITH_ITS` will be used in all cases where we have ITS, and `CUT_RANDOM_FRACTION_MCH_NO_ITS` if we don't.